### ISSUE TITLE
Unable to fetch legacy rep data - invalid URL

### ISSUE THREAD

**Issue by yani** [xadamxk/HFX2.0#268]
```
GET https://raw.githubusercontent.com/xadamxk/HFX2.0/feature/legacy-reputation/reputationArchive.json 404
HFX.js:1044 [HFX] Failed to fetch LegacyReputation data.
```

Link points to a removed branch.

For security reasons I suggest pointing it not to `master` but to a tag: `https://github.com/xadamxk/HFX2.0/blob/v2.6.0/reputationArchive.json`

**Comment by xadamxk** [comment]
Seems like you are using the extension from source and not what is deployed via web store?
`develop` is working - master is missing the fix that develop has until I merge changes into it.

**Comment by yani** [comment]
No I just installed it using the web store.

**Comment by xadamxk** [comment]
Ahh, I'll push a fix for that along with a handful of other tweaks in the next few days. Thanks for raising awareness on this.

**Comment by xadamxk** [comment]
Will be fixed in #275 thanks again :)
